### PR TITLE
Feature/batch

### PR DESCRIPTION
**Pull request by w1shmast0ur**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const {json, send} = require('micro')
+const { json, send } = require('micro')
```
Comment (elnygren) [inline server.js]: @w1shmast0ur can you configure your editor in such a way that we don't need to change existing codestyle on the fly :)

File: server.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 const FCM = require('fcm-node')
-require('console-stamp')(console, {pattern: 'dd/mm/yyyy HH:MM:ss.l'})
+require('console-stamp')(console, { pattern: 'dd/mm/yyyy HH:MM:ss.l' })
```
Comment (elnygren) [inline server.js]: ^

File: server.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const handleErrors = fn => async (req, res) => {
     return await fn(req, res)
   } catch (err) {
     console.error(err.stack)
-    send(res, 500, {error: 'internal-error', message: 'Internal server error'})
+    send(res, 500, { error: 'internal-error', message: 'Internal server error' })
```
Comment (elnygren) [inline server.js]: ^

File: server.js [hunk 4]
```diff
@@ -31,12 +31,28 @@ function fcmCallback(err, res) {
 
   const errors = res.results.filter(obj => ('error' in obj))
   if (errors.length) {
-    errors.forEach(obj => {console.error('error:', obj.error)})
+    errors.forEach(obj => { console.error('error:', obj.error) })
```
Comment (elnygren) [inline server.js]: ^

File: server.js (hunk 1 above)
Comment (elnygren) [reply server.js]: you can  add a `.editorconfig` or similar to the project if that helps !

Comment (w1shmast0ur) [reply server.js]: - [x] check

File: server.js (hunk 2 above)
Comment (w1shmast0ur) [reply server.js]: - [x] check

File: server.js (hunk 3 above)
Comment (w1shmast0ur) [reply server.js]: - [x] check

File: server.js (hunk 4 above)
Comment (w1shmast0ur) [reply server.js]: - [x] check

**Pull request merged**